_includes/pbc/all/install-features/202304.0/install-the-packaging-units-feature.md

```diff
@@ -0,0 +1,1063 @@
+        ];
+    }
+}
+```
+
+**src/Pyz/Client/PersistentCart/PersistentCartDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Client\PersistentCart;
+
+use Spryker\Client\PersistentCart\PersistentCartDependencyProvider as SprykerPersistentCartDependencyProvider;
+use Spryker\Client\ProductPackagingUnit\Plugin\PersistentCartExtension\ProductPackagingUnitAmountPersistentCartChangeExpanderPlugin;
+
+class PersistentCartDependencyProvider extends SprykerPersistentCartDependencyProvider
+{
+    /**
+     * @return \Spryker\Client\PersistentCartExtension\Dependency\Plugin\PersistentCartChangeExpanderPluginInterface[]
+     */
+    protected function getChangeRequestExtendPlugins(): array
+    {
+        return [
+            new ProductPackagingUnitAmountPersistentCartChangeExpanderPlugin(), #ProductPackagingUnit
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Add an item with packaging units to cart.
+
+*     Check if a packaging unit can be found for an item.
+*     Check if the `amount`, `amountSalesUnit`, `amountLeadProduct` and `ProductPackagingUnit` fields in the `ItemTransfer` properties get fully populated.
+*     Check if the amount restriction works as expected.
+*     Check if availability is validated respectfully according to your lead product's and packaging unit's configuration.
+*     Check if item grouping in cart works as expected.
+*     Check if variable amount changes affect unit prices in the `ItemTransfer` properties.
+*     Check if quantity and amount are merged correctly when the group key matches.
+
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
+
+Go through the checkout workflow and make an order>.
+* Check if the stock is modified respectfully according to your lead product's and packaging unit's configuration.
+* Check if the following fields in the `spy_sales_order_item` table are saved:
+  * `amount`
+  * `amount_sku`
+  * `amount_measurement_unit_name`
+  * `amount_measurement_unit_code`
+  * `amount_measurement_unit_precision`
+  * `amount_measurement_unit_conversion`
+  * `amount_base_measurement_unit_name`
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
+Go to the Zed UI Sales overview and check the order.
+- Verify if the correct sales unit is displayed.
+- Verify if the correct amount is displayed per sales order item.
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
+
+Make sure that abstract products which have packaging units available don’t have `add_to_cart_sku` field at Elasticsearch document.
+
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
+
+Make sure that every order item from `SalesFacade::getOrderItems()` results contains packaging units data: `ItemTransfer.amountLeadProduct` and `ItemTransfer.amountSalesUnit` are set for the order items that have packaging units.
+
+{% endinfo_block %}
+
+
+## Install feature frontend
```

Comment:
* We have to add the missing plugin:
QuickOrderItemDefaultPackagingUnitExpanderPlugin